u6-spas/src/views/Converter.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <table>
+    <tr>
+      <td><input v-model="input" type="number" /></td>
+      <td>=</td>
+      <td>
+        <span class="output">{{ output.toFixed(2) }}</span>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <select v-model="inputUnit">
+          <option v-for="unit in units" :key="unit.id" :value="unit.id">
+            {{ unit.label }}
+          </option>
+        </select>
+      </td>
+      <td class="fake-link">swap</td>
+      <td>
+        <select v-model="outputUnit">
+          <option v-for="unit in units" :key="unit.id" :value="unit.id">
+            {{ unit.label }}
+          </option>
+        </select>
```

Comment:
This is a good example for discussing the uniqueness requirements of `:key`. You have 2 v-fors that both use the same keys. In class I mentioned that the key must be globally unique. Actually I was wrong... and this is an example where the keys do not conflict! Keys must only be unique if the v-for elements are siblings...

**Ok** (the v-fors have different parent elements):
```vue
<select v-model="inputUnit">
  <option v-for="unit in units" :key="unit.id" :value="unit.id">{{ unit.label }}</option>
</select>
<select v-model="outputUnit">
  <option v-for="unit in units" :key="unit.id" :value="unit.id">{{ unit.label }}</option>
</select>
```
**Not ok** (the v-fors have the same parent element & the same keys):
```vue
<div>
  <div v-for="unit in units" :key="unit.id" :value="unit.id">
    <a :href="`#${unit.label}`">Jump to {{ unit.label }}</a>
  </div>
  <div v-for="unit in units" :key="unit.id" :value="unit.id">
    <h1 :id="unit.id">{{ unit.label }}</h1>
    <p>{{ unit.description }}</p>
  </div>
</div>

```